alacritty/src/renderer/rects.rs

```diff
@@ -184,3 +184,13 @@ impl RenderLines {
         }
     }
 }
+
+#[inline]
+fn into_cell_end(cell: &RenderableCell) -> Point {
+    let mut end: Point = cell.into();
+    if cell.flags.contains(Flags::WIDE_CHAR) {
+        end.col += 1;
+    }
+
+    end
+}
```

Comment:
One way or another this gets invoked anyways. Can't we just calculate the point after the `!cell.flags.contains(flag)` check and then always use that? Just as efficient and sounds much cleaner.